### PR TITLE
show monitor name instead of fubar annotations

### PR DESCRIPTION
**Pull request by dbrinegar**
since the UI doesn't let one see or edit the annotations

### CODE REVIEW

File: enterprise-suite/alertmanager/slack.tmpl [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {{ define "title.slack" }}{{ .GroupLabels.namespace}}/{{ .GroupLabels.es_workload }} [{{ .Status | toUpper }}][{{ .CommonLabels.severity | toUpper }}]{{ end }}
-{{ define "text.slack" }}{{ range $index, $element := .Alerts }}_{{ .Annotations.summary }}_
-    {{ .Annotations.description }}
+{{ define "text.slack" }}{{ range $index, $element := .Alerts }}_{{ .CommonLabels.name }}_
```
Comment (alanngai) [inline enterprise-suite/alertmanager/slack.tmpl]: I assume .CommonLabels.name is the monitor name?  If so, lgtm

**Comment by jsravn** [comment]
@dbrinegar we really wanted to drop the alert description/summary? I thought we just wanted to add the alert name.

**Comment by dbrinegar** [comment]
The UI doesn't show or allow editing of those fields, and the Monitor API doesn't provide generic defaults, so their content at the moment is random and probably not suitable. I think if the UI were to submit them blank and the Monitor API were to come up with generic defaults suitable for the kind of monitor, it would be reasonable to include those messages in the alerts.

File: enterprise-suite/alertmanager/slack.tmpl (hunk 1 above)
Comment (jsravn) [reply enterprise-suite/alertmanager/slack.tmpl]: Seems like it's not? At least, @srikanthops is having issues with it.

Comment (jsravn) [reply enterprise-suite/alertmanager/slack.tmpl]: Oh I see the issue - .CommonLabels isn't available in the range. It should be `.Labels.name`.

**Pull request merged**